### DISCUSSION TITLE
filter by exif IFD

### DISCUSSION THREAD

**Discussion by l-leroux** [deckerst/aves#842] [Ideas]
Hi !
Thank you for this beautiful and useful app !
I manage images from a unistellar telescope which fill automatic information in EXIF IFD0.
Is it possible to filter a collection or a directory by one or more of theses tag ?
I didn't find how.
It would be very smart !
Best regards,

**Comment by deckerst** [comment]
Likely similar need as #424.

Out of curiosity, would you be able to provide a sample telescope image?

**Comment by l-leroux** [comment]
There is one them here : ![eVscope-20230716-210439](https://github.com/deckerst/aves/assets/73694741/368aadeb-2774-4a12-8657-dcdd07e40e42)
The most interesting fields seem to be "image description" in exif ifd0, and maybe "exposure time"  in exif subifd.

By the way, if you're interested in great images, visit unistellar.com 😁

**Reply by l-leroux** [reply]
For example, the goal is to compare several image of the same object taken from different places or at different moment (of with different exposure time, in order to filter only the most beautiful one).

**Reply by deckerst** [reply]
I understand that parsing Exif description could be useful, but for what you describe tags are perfect. I have other astronomy pictures where they tagged the image with the object taken, the telescope or the probe that took it, etc. It's flexible and better organized than raw text.

**Reply by l-leroux** [reply]
But tags are written by the user, aren't they ?
Exif are written by the telescope... By a full nigth of observation, it could represent dozen of images... I used xnview with tags before, but it was a long work to tag every image with information that already exists in the exif...

**Reply by deckerst** [reply]
The telescope could write XMP tags just like it writes Exif fields.

**Reply by l-leroux** [reply]
In theory, yes. I've just asked them to do so, but I'm not confident...
